### PR TITLE
Contact redesign oct4

### PR DESCRIPTION
**Pull request by brentbagapuro**

### CODE REVIEW

File: pages/contact.js [hunk 1]
```diff
@@ -1,58 +1,51 @@
...
-      '&:hover': {
-        backgroundColor: theme.palette.primary.main,
-      },
-    },
-  },
-}))
-
-const Contact = () => {
-  const classes = useStyles()
+const CotnactPage = () => {
```
Comment (dccabs) [inline pages/contact.js:9]: Please correct spelling here.

**Comment by dccabs** [comment]
Please replace the dummy information with what we have on the housecall contact us form now

Screen Shot 2022-10-04 at 1 57 46 PM

Please replace the MUI snackbar with a notification modal.

Please put in a modal if there is an error with the submission.

Right now I'm still seeing default MUI fonts.  Basically we want to strip MUI out of this application on the customer side.  So ideally we can figure out a way to strip out all MUI.  We do want to keep it on the admin side for now.

Thanks 

-dan

**Review by dccabs — CHANGES_REQUESTED** [review]
Hi Brent, please see comments for changes. Thank you.

File: pages/contact.js [hunk 2]
```diff
@@ -1,79 +1,67 @@
...
-      }
     })
+      .then((res) => {
+        if (res.ok) {
+          setFormError(null)
+        }
+      })
+      .catch((err) => setFormError(err.message))
+
+    setOpenDialog(true)
```
Comment (dccabs) [inline pages/contact.js:64]: This should be put in the success portion of the fetch request.  Right now it calls before the submission is made.

Comment (dccabs) [reply pages/contact.js:64]: Can we add a loading indicator that the message is sending.

**Comment by brentbagapuro** [comment]
Fixed all merge issues. This branch is ready to merge to housecall_tailwinds_ui

**Pull request merged**